configs/datasets/vln/mp3d_r2r.yaml

```
@@ -0,0 +1,5 @@
+DATASET:
+  TYPE: R2RVLN-v1
+  SPLIT: train
+  DATA_PATH: data/R2R/habitat_R2R_{split}.json
```

To follow naming dataset files naming convention I would advice next naming:
```suggestion
  DATA_PATH:   data/datasets/r2r/{split}/{split}.json.gz
```